app/src/proto-recs/proto.css

```
@@ -0,0 +1,98 @@
+  border-color: rgba(255, 255, 255, .4);
+  background: rgba(255, 255, 255, .1);
+}
+
+/* ── Variant motion ────────────────────────────────────────────────────── */
+/* Entrances only, transform/opacity only, ease-out, all well under 300ms.
+   base.css's reduced-motion block collapses every one of them. */
+
+/* Beacon's allocation ribbon: each band wipes out from the start edge. The
+   origin has to follow the writing direction — a ribbon that always grows
+   from the left grows backwards in Hebrew. */
+@keyframes protoRibbon {
```

Stylelint (17.14.0)

[error] Expected keyframe name "protoRibbon" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)